unrasterize/classes.py

```diff
@@ -126,7 +126,7 @@ def _sort_pixels(band):
         )
 
     @staticmethod
-    def _reassign_pixel_values(band, pixels, raw_pixel_values=[]):
+    def _reassign_pixel_values(threshold, band, pixels, raw_pixel_values=[]):
```

Comment:
Nit: I have a slight preference for the argument order `band, pixels, threshold, ...`